docusaurus/docs/available-scripts.md

```diff
@@ -22,6 +22,10 @@ Builds the app for production to the `build` folder. It correctly bundles React
 
 The build is minified and the filenames include the hashes. See the [production build](production-build.md) section for more information.
 
+React DOM automatically supports profiling in development mode for v16.5+, but since profiling adds some small
```

Comment:
I think these docs should actually be on this page: https://create-react-app.dev/docs/production-build

Reply:
Should there be a reference to it in the script's document maybe or some kind of short mention?

Reply:
Sure, we can mention it and link to the more detailed docs.

Reply:
Also if I add the documentation in the production-build should it just be another header/section like "Static File Caching"

Reply:
Yes.

Reply:
Great, thank you! I moved it and added a brief reference to the new documentation in the same section in available scripts.